### PR TITLE
Sync up VODE90 with VODE/BS retry strategy

### PR DESCRIPTION
**Pull request by maxpkatz**

### CODE REVIEW

File: integration/Make.package [hunk 1]
```diff
@@ -12,7 +12,8 @@ else ifeq ($(INTEGRATOR_DIR),BS)
 else ifeq ($(INTEGRATOR_DIR),VBDF)
   INTEGRATOR_NUM := 2
 else ifeq ($(INTEGRATOR_DIR),VODE90)
-  INTEGRATOR_NUM := 3
+  # VODE90 is a drop-in replacement for VODE, treat it the same way.
+  INTEGRATOR_NUM := 0
```
Comment (zingale) [inline integration/Make.package]: I think we have some places in Castro where we check the value of this in if-defs, so changing 3 to 0 might break code.

Comment (maxpkatz) [reply integration/Make.package]: Renumbered to 3

**Comment by zingale** [comment]
can you add a description to the CHANGES?

**Comment by zingale** [comment]
also, there is a merge conflict -- is the resolution to just remove the old code?

**Pull request merged**